src/components/RingGraph/index.tsx

```diff
@@ -0,0 +1,100 @@
+
+import React from "react";
+import {View, Text} from 'react-native';
+import {styles} from './styles'
+import Svg, { G, Circle } from "react-native-svg";
+
+const RingGraph = () => {
+
+const radius = 65;
+const borderTest = 10;
+const circleCircumference = 2*Math.PI*radius;
+//Need to fix how data is added here
+const BTC = 1000;
+const ETH = 1000;
+const SOL = 1400;
+const total = BTC+ETH+SOL;
+
+const BTCPercentage = (BTC/total)*100;
+const ETHPercentage = (ETH/total)*100;
+const SOLPercentage = (SOL/total)*100;
+
+const BTCStrokeDashoffset =
+    (circleCircumference - (circleCircumference * BTCPercentage)) / 100;
+  const ETHStrokeDashoffset =
+    (circleCircumference - (circleCircumference * ETHPercentage)) / 100;
+  const SOLStrokeDashoffset =
+    (circleCircumference - (circleCircumference * SOLPercentage)) / 100;
+
+const BTCAngle = (BTC/total)*360;
+const ETHAngle = (ETH/total)*360;
+const SOLAngle = (SOL/total)*360;
```

Comment:
Let's turn these calculations into a function so that we can reuse the function for however many coins there will be.